simpleLooping/test_removeSpaceAtEnd.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+from removeSpaceAtEnd import removeSpaceAtEnd
+
+class TestRemoveSpaceAtEnd(unittest.TestCase):
+    def test_emptyList(self):
+        self.assertEqual(removeSpaceAtEnd(""), "")
+
+    def test_spaceAtTheEndInput(self):
```

Comment:
What is “spaceAtEnd” mean? The other problem you make it so nice with “hasSpaceAtEnd”, so why the different?

Reply:
ok

Reply:
Sorry, I don’t understand what you mean by “ok”? Do you agree, not agree, or planning to fix?

Reply:
Hello? Did you have a look on this one?

Reply:
Please review this.